Backend/HummingWings/api/models/booking_holder.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+class BookingHolder(models.Model):
+    email = models.EmailField(verbose_name="Correo")
+    cellphone = models.CharField(max_length=255, verbose_name="Celular")
+    tickets_pending = models.ManyToManyField('api.Ticket', related_name='booking_holders', blank=True)
```

Comment:
Este campo debe ser One to many. Es decir, un booking holder puede tener muchos tickets, no al reves

Reply:
Sapo